pkg/storage/sds/edvdocumentprocessor/edvdocumentprocessor.go

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package edvdocumentprocessor
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/storage/sds"
+)
+
+// EDVDocumentProcessor represents a type that can encrypt and decrypt between
+// Structured Documents and Encrypted Documents.
+type EDVDocumentProcessor interface {
```

Comment:
all packages, files and interface start with edv - there is a lot of stutter

Reply:
Moved all the "edv" files within a folder and removed that prefix.